### DISCUSSION TITLE
Issues with Copilot in Jetbrains IDEs

### DISCUSSION THREAD

**Discussion by p4nu** [community/community#7344] [Copilot Conversations] [answered]
Is there a known issues page for Copilot in Jetbrains IDEs?

**Comment by nbrugger-tgm** [comment]
One unexpected behaviour is that the TAB key sometimes performs the intellij action (for example jump to next marked place/next line) instead of completing the previewed code.

Reproduce:
(java)
```java
void main(){
      callSomeMethodWithArguments( <cursor here>
}
```

Copilot will corectly suggest something like `someGlobalValue);` but tab just jumps around

**Reply by jansorg** [reply]
This should be fixed with the latest update, 1.0.3. Could you try that one, please?

**Comment by jansorg** [comment, marked as answer]
There's currently no such page. 
Are there any problems in particular?
